### DISCUSSION TITLE
Implement Window Preview

### DISCUSSION THREAD

**Discussion by c-leri** [linux-is-awesome/gnome_extension_rocketbar#7] [Ideas]
It would be nice if you implemented the window preview from [App Icons Taskbar](https://gitlab.com/AndrewZaech/aztaskbar)

**Comment by ChepKun** [comment]
Actually I don't have any plans to implement this feature, because I hate it personally :D I go with 'advanced' tooltips instead.

**Reply by c-leri** [reply]
Oh ok then
